packages/gatsby/src/utils/__tests__/browserslist.js

```diff
@@ -28,7 +28,14 @@ describe(`browserslist`, () => {
 
     const list = getBrowsersList(BASE)
 
-    expect(list).toEqual([`>0.25%`, `not dead`])
+    if (_CFLAGS_.GATSBY_MAJOR === `5`) {
```

Comment:
I'm not sure we'll ever get to this point in the tests. Given that CFLAGS are environmental set by the compiler, they won't exist in the tests. I could be wrong, but either way, we won't ever be testing both cases.

Can we add a `_CFLAGS_` mock to the setup/teardown? This might need some tweaking, but something like:

```
let originalCflags
beforeEach(() => {
  originalCflags = _CFLAGS_
  
  # TODO set up mock vars, set them in tests, etc
  _CFLAGS_ = {...}
})

afterEach(() => {
  _CFLAGS_ = originalCflags
})
```

Reply:
We do pass in CFLAGS flags to test with `babel-preset-gatsby-package`.

My thinking is that we'll eventually remove the else block since we'll never hit that when the compile flags become 5

cc - @wardpeet

Reply:
Ahh ok. I won't push too hard on it because the logic is simple, but I'd prefer to have full logic coverage even before the Gatsby 5 launch.